website/versioned_docs/version-0.19.0/getting_started.md

```diff
@@ -0,0 +1,8 @@
+---
+id: version-0.19.0-getting_started
+title: Getting Started
+sidebar_label: Getting Started
+original_id: getting_started
+---
+
+This page will help you set up React Native Elements with your app.
```

Comment:
Maybe add back the installation instructions

Reply:
Definitely